RadixWallet/Core/SharedModels/Assets/OnLedgerEntity.swift

```diff
@@ -555,7 +562,7 @@ extension OnLedgerEntity.OnLedgerAccount {
 		var debugDescription: String {
 			"""
 			\(validatorAddress.formatted())
-			staked: \(stakeUnitResource?.amount.nominalAmount.formatted() ?? "NONE")
+			staked: \(stakeUnitResource?.amount.debugDescription ?? "NONE")
```

Comment:
I know we are calling `.debugDescription` in a LOT of places, but it is discouraged by Apple:
https://developer.apple.com/documentation/swift/customdebugstringconvertible/debugdescription#discussion

> Calling this property directly is discouraged. Instead, convert an instance of any type to a string by using the String(reflecting:) initializer. This initializer works with any type, and uses the custom debugDescription property for types that conform to CustomDebugStringConvertible:

So perhaps we ought for new code use `String(reflecting:)` instead?